subsys/logging/log_backend_native.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+#include <logging/log_backend_native.h>
+#include <logging/log_core.h>
+#include <logging/log_msg.h>
+#include <logging/log_output.h>
+#include <device.h>
+#include <uart.h>
+
+static u8_t buf;
+
+int char_out(u8_t *data, size_t length, void *ctx)
+{
+	for (size_t i = 0; i < length; i++) {
+		putchar(data[i]);
+	}
+
+	return length;
+}
+
+static struct log_output_ctx ctx = {
+	.func = char_out,
+	.data = &buf,
+	.length = 1,
```

Comment:
yes, so this is a copy-paste of uart backend which is very simple one (polling) and might be reworked in the future. Here, it may make sense to have the buffer bigger if block operations are faster.